templates/typescript-testing/tsconfig.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
is this aligned with the current hello world example?

Reply:
Yes, the file is aligned